scripts/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+var slideshow = [];
+
+function Slide (slideObject) {
```

Comment:
Or are you planning on making your individual projects a slideshow? If so, I would just focus on semantic naming and your naming conventions in general. 
A note I wrote earlier: You need an object constructor to add individual projects to your portfolio page. You have a constructor, but it looks like it is for slides(by its name) yet the data that it takes in looks like it would be project info. I would rework this so the naming and its functionality are similar.